web/docs/guides/database/functions.mdx

```diff
@@ -0,0 +1,312 @@
+```
+
+</TabItem>
+<TabItem value="Data">
+
+<h4>Planets</h4>
+
+| id    | name | 
+| ----  | ---- | 
+|    1 | Tattoine | 
+|    2 | Alderaan  | 
+|    3 | Kashyyyk | 
+
+<h4>People</h4>
+
+| id    | name | planet_id |
+| ----  | ---- | ---- | 
+|    1 | Anakin Skywalker | 1 | 
+|    2 | Luke Skywalker  | 1 | 
+|    3 | Princess Leia | 2 | 
+|    4 | Chewbacca | 3 | 
+
+</TabItem>
+
+</Tabs>
+
+### Basic function
+
+A basic example which returns a string "hello world".
+
+```sql
+create or replace function hello_world() -- 1
+returns text -- 2
+language sql -- 3
+as $$  -- 4
+  select 'hello world';  -- 5
+$$; --6
+
+```
+
+#### Details
+
+<details>
+<summary>Show/Hide</summary>
+  
+At it's most basic a function has the following parts:
+
+1. `create or replace function hello_world()`
+- The function declaration, where `hello_world` is the name of the function.
+- You can use either `create` when creating a new fucntion or `replace` when replacing an existing function. Or you can use `create or replace` together to handle either.
+2. `returns text`
+- The type of data that the function returns. If it returns nothing, you can `return void`. 
+3. `language sql`
+- The language used inside the function body. This can also be a procedural language: `plpgsql`, `plv8`, `plpython`, etc.
+4. `as $$`
+- The function wrapper. Anything enclosed inside the `$$` symbols will be part of the function body.
+5. `select 'hello world';`
+- A simple function body. The final `select` statement inside a function body will be returned if there are no statements following it.
+6. `$$;`
+- The closing symbols of the funciton wrapper.
+
+</details>
+
+#### Execute
+
+<Tabs
+defaultValue="SQL"
+values={[
+  {label: 'SQL', value: 'SQL'},
+  {label: 'JS', value: 'JS'},
+  {label: 'Dart', value: 'Dart'},
+]}>
+
+<TabItem value="SQL">
+
+```sql
+select hello_world();
+```
+
+</TabItem>
+<TabItem value="JS">
+
+```js 
+const { data, error } = supabase
+  .rpc('hello_world')
+```
+
+Reference: [rpc()](/docs/reference/javascript/rpc)
+
+</TabItem>
+<TabItem value="Dart">
+
+```dart 
+final res = await supabase
+  .rpc('hello_world')
+  .execute();
+```
+
+Reference: [rpc()](/docs/reference/dart/rpc)
+
+</TabItem>
+
+</Tabs>
+
+
+### Return table sets
+
+A basic example which return all the `planets` in our example data set.
+
+```sql
+create or replace function get_planets() 
+returns setof planets
+language sql
+as $$
+  select * from planets;
+$$;
+```
+
+#### Execute 
+
+Because this function returns a table set, we can also apply filters and selectors. For exaple, if we only wanted the `name` of the first planet:
+
+<Tabs
+defaultValue="SQL"
+values={[
+  {label: 'SQL', value: 'SQL'},
+  {label: 'JS', value: 'JS'},
+  {label: 'Dart', value: 'Dart'},
+]}>
+
+<TabItem value="SQL">
+
+```sql
+select name
+from get_planets()
+where id = 1;
+```
+
+</TabItem>
+<TabItem value="JS">
+
+```js 
+const { data, error } = supabase
+  .rpc('get_planets')
+  .eq('id', 1)
+```
```

Comment:
I think you might have missed the `.select("name")` call to select just the name field like the description specifies.